numpy/core/_exceptions.py

```diff
@@ -122,21 +122,105 @@ class TooHardError(RuntimeError):
 
 @set_module('numpy')
 class AxisError(ValueError, IndexError):
-    """ Axis supplied was invalid. """
+    """Axis supplied was invalid.
+
+    Raised whenever an ``axis`` parameter is specified that is larger than
+    the number of array dimensions.
+
+    .. versionadded:: 1.22
```

Comment:
```suggestion
    .. versionchanged:: 1.22
```
unless you want to put this in the `attributes` section